README.md

```diff
@@ -1,13 +1,19 @@
-# Project Name
+# Guess Who Game
 
-Replace this readme with your own information about your project.
+This is a web-based implementation of the classic Guess Who board game, where the player has to guess a secret person from a board with different characters. 
+The project was built using HTML, CSS, and JavaScript, and includes the following features:
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The board with characters is generated dynamically when the website is loaded.
```

Comment:
Very nice README, I liked reading it. Although it says to keep it brief I guess no one minds a litle bit of details to the story :)